rkt/image/fetchers/common.go

```diff
@@ -0,0 +1,49 @@
+package fetchers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type InsecureOpts struct {
+	AllowHTTP      bool `json:"allow_http"`
+	SkipTLSCheck   bool `json:"skip_tls_check"`
+	SkipImageCheck bool `json:"skip_image_check"`
+}
+
+type FetcherConfig struct {
+	Scheme        string                         `json:"scheme"`
+	Name          string                         `json:"name"`
+	InsecureOpts  *InsecureOpts                  `json:"insecure"`
+	Debug         bool                           `json:"debug"`
+	Headers       map[string]map[string][]string `json:"headers"`
+	OutputACIPath string                         `json:"output_aci_path"`
+	OutputASCPath string                         `json:"output_asc_path"`
+}
+
+type FetcherResult struct {
+	Latest    bool   `json:"latest"`
+	ETag      string `json:"etag"`
+	MaxAge    int    `json:"max_age"`
+	UseCached bool   `json:"use_cached"`
+}
+
+func GetConfigFromStdin() (*FetcherConfig, error) {
```

Comment:
I think he doesn't mean the methods themselves, just the function naming; having `Get` as part of a method name is non-idiomatic go.

While we're on idiomatic, there's also some stuttering in `fetchers.FetcherConfig` etc.